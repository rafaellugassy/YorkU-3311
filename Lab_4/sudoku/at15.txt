set_number(3, 2, 1) 	-- 1
set_number(2, 2, 2) 	-- 2
set_number(1, 3, 3) 	-- 3
set_number(3, 3, 4) 	-- 4

start_game 				-- 5

put_number(4, 3, 2) 	-- 6
put_number(2, 3, 1)		-- 7

undo -- back to after executing 6
undo -- back to after executing 5
undo -- back to after executing 4
undo -- back to after executing 3
undo -- back to after executing 2
undo -- back to after executing 1
undo -- back initial state

undo -- error: nothing to undo

redo -- back to after executing 1
redo -- back to after executing 2
redo -- back to after executing 3
redo -- back to after executing 4
redo -- back to after executing 5
redo -- back to after executing 6
redo -- back to after executing 7

redo -- error: nothing to redo