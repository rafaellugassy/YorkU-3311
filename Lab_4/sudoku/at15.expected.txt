  Setting up a new grid
  0   0   0   0
  0   0   0   0
  0   0   0   0
  0   0   0   0
->set_number(3,2,1)
  Setting up a new grid
  0   0   0   0
  3   0   0   0
  0   0   0   0
  0   0   0   0
->set_number(2,2,2)
  Setting up a new grid
  0   0   0   0
  3   2   0   0
  0   0   0   0
  0   0   0   0
->set_number(1,3,3)
  Setting up a new grid
  0   0   0   0
  3   2   0   0
  0   0   1   0
  0   0   0   0
->set_number(3,3,4)
  Setting up a new grid
  0   0   0   0
  3   2   0   0
  0   0   1   3
  0   0   0   0
->start_game
  Current game is over: No
  0   0   0   0
  3   2   0   0
  0   0   1   3
  0   0   0   0
->put_number(4,3,2)
  Current game is over: No
  0   0   0   0
  3   2   0   0
  0   4   1   3
  0   0   0   0
->put_number(2,3,1)
  Current game is over: No
  0   0   0   0
  3   2   0   0
  2   4   1   3
  0   0   0   0
->undo
  Current game is over: No
  0   0   0   0
  3   2   0   0
  0   4   1   3
  0   0   0   0
->undo
  Current game is over: No
  0   0   0   0
  3   2   0   0
  0   0   1   3
  0   0   0   0
->undo
  Setting up a new grid
  0   0   0   0
  3   2   0   0
  0   0   1   3
  0   0   0   0
->undo
  Setting up a new grid
  0   0   0   0
  3   2   0   0
  0   0   1   0
  0   0   0   0
->undo
  Setting up a new grid
  0   0   0   0
  3   2   0   0
  0   0   0   0
  0   0   0   0
->undo
  Setting up a new grid
  0   0   0   0
  3   0   0   0
  0   0   0   0
  0   0   0   0
->undo
  Setting up a new grid
  0   0   0   0
  0   0   0   0
  0   0   0   0
  0   0   0   0
->undo
  Error: Nothing to undo
  0   0   0   0
  0   0   0   0
  0   0   0   0
  0   0   0   0
->redo
  Setting up a new grid
  0   0   0   0
  3   0   0   0
  0   0   0   0
  0   0   0   0
->redo
  Setting up a new grid
  0   0   0   0
  3   2   0   0
  0   0   0   0
  0   0   0   0
->redo
  Setting up a new grid
  0   0   0   0
  3   2   0   0
  0   0   1   0
  0   0   0   0
->redo
  Setting up a new grid
  0   0   0   0
  3   2   0   0
  0   0   1   3
  0   0   0   0
->redo
  Current game is over: No
  0   0   0   0
  3   2   0   0
  0   0   1   3
  0   0   0   0
->redo
  Current game is over: No
  0   0   0   0
  3   2   0   0
  0   4   1   3
  0   0   0   0
->redo
  Current game is over: No
  0   0   0   0
  3   2   0   0
  2   4   1   3
  0   0   0   0
->redo
  Error: Nothing to redo
  0   0   0   0
  3   2   0   0
  2   4   1   3
  0   0   0   0
